src/cmd/flash.rs

```diff
@@ -64,7 +59,7 @@ fn determine_board_to_flash(
                     board_from_dut
                 ));
             }
-            Ok(board_from_arg)
+            Ok(board_from_dut)
```

Comment:
I just want to double check this works as intended when we pass something with -kernelnext.

Reply:
Yeah, I confirmed it works 
```
$ lium flash --usb --board brya-kernelnext 
13:26:36.819: INFO: Preparing to image the removable device 
Removable device(s) found. Please select/confirm to continue:
  [0]: SanDisk USB Extreme Pro  (/dev/sda)
Please choose an option [0-0]: 
```

Reply:
Ah, I see. This must use `board_from_arg` instead of `board_from_dut`, because e.g. -kernelnext variant will be only in the `board_from_arg`. Thanks!